src/client/Microsoft.Identity.Client/MsalClientException.cs

```diff
@@ -19,7 +19,8 @@ public class MsalClientException : MsalException
         /// <param name="errorCode">
         /// The error code returned by the service or generated by client. This is the code you can rely on
         /// for exception handling.</param>
-        public MsalClientException(string errorCode) : base(errorCode)
+        /// <param name="isRetryable">The boolean to indicate whether to perform retry operation for the exception.</param>
+        public MsalClientException(string errorCode, bool isRetryable = false) : base(errorCode, isRetryable)
```

Comment:
Personally I like the ctor approach more because it enables read-only properties and immutability. Even if immutability may not matter that much in this case, it makes reading / reasoning about code simpler.

Reply:
Updated as suggested by Sameer.